src/draw/vg_lite/lv_draw_vg_lite_label.c

```diff
@@ -223,7 +223,7 @@ static void draw_letter_bitmap(lv_draw_vg_lite_unit_t * u, const lv_draw_glyph_d
      * You need to wait for the GPU to finish using the buffer before releasing it.
      * Later, use the font cache for management to improve efficiency.
      */
-    lv_vg_lite_finish(u);
+    // lv_vg_lite_finish(u);
```

Comment:
Accidentally left here commented (because of debugging), or it is really not needed?
In latter case you could remove it, git history will have it anyway.